package/CHANGELOG

```diff
@@ -30,6 +30,7 @@ Fixes
   * Fix atom charge reading in PDBQT parser (Issue #4282, PR #4283)
 
 Enhancements
+  * Document the usage of NoDataError in it's docstring (Issue #3901, PR #4359)
```

Comment:
```suggestion
  * Document the usage of NoDataError in its docstring (Issue #3901, PR #4359)
```

Reply:
will update it in the following commits.